src/reducers/entities/cloud.ts

```diff
@@ -5,7 +5,18 @@ import {combineReducers} from 'redux';
 import {CloudTypes} from 'action_types';
 
 import {GenericAction} from 'types/actions';
-import {CloudCustomer} from 'types/cloud';
+import {Product, Subscription, CloudCustomer} from 'types/cloud';
+import {Dictionary} from 'types/utilities';
+
+function subscription(state: Subscription | null = null, action: GenericAction) {
```

Comment:
It's annoying that we need to use `| null` here because of redux. It would be nice to be able to just use `undefined` everywhere.